Learning Team Agenda - September 17th, 2020 (#337)
emberjs/core-notes: learning-team/2020-09/2020-09-17.md

# September 17, 2020

Facilitator & Note Taker: 

## Attendees

Add yourself to the list if you attend and check the box!

### Core Team
- [ ] Amy Lam (AL)
- [ ] Chris Manson (CM)
- [ ] Isaac Lee (IL)
- [ ] Jared Galanis (JG)
- [ ] Jen Weber (JW)
- [ ] Jessica Jordan (JJ)
- [ ] Kenneth Larsen (KL)
- [ ] Melanie Sumner (MS)
- [ ] Ricardo Mendes (RM)
- [ ] Robert Wagner (RW)

### Guests
- [] Your name here (INITIALS)

## Weekly review

### Review action items (weekly)
See: [Action items board](https://github.com/orgs/ember-learn/projects/47)

### Review Initiatives board (every other week)
See: [Initiatives board](https://github.com/orgs/ember-learn/projects/19)

### Review PR queue
See: [tracking sheet](https://docs.google.com/spreadsheets/d/1sPyN9z9wZMpTNwqCfa6R9QSPZkIW4iQd-H4gZC7ILLk/edit#gid=2035777454)

## Topics

<!-- If you would like to add a topic to the agenda please add a suggestion to the PR using the following format: -->
<!-- ### Your topic (INITIALS, expected duration in minutes) -->
<!-- replace with topic -->
<!-- replace with topic -->
<!-- replace with topic -->
<!-- replace with topic -->

> <!-- replace with topic -->

Reviewer: ### Feedback for Ember Guides (IL, 10 min)

In the Ember Guides, we use the `+` and `-` to indicate what code have been added or deleted. For example, in the [Super Rentals Tutorial](https://guides.emberjs.com/release/tutorial/part-1/orientation/#toc_editing-files-and-live-reload) (and other pages), we might see,

<img width="480" alt="Screen Shot 2020-09-16 at 4 05 14 PM" src="https://user-images.githubusercontent.com/16869656/93392480-7995bc00-f836-11ea-953e-d0161a7538f8.png">

A developer provided me feedback that 2 interns that they were mentoring weren't sure how to read such code. The developer explained to me that the interns weren't yet familiar with diff tools that we might see on GitHub.

For more complex code (think about a large code review that you might have had to do), I think it's also possible that showing many changes with `+` or `-` may not be the best way to convey the changes.

I wondered if we can **update the code block component** to either:

- Provide a help button that, when clicked, explains how to read code with `+` and `-`
- Provide an option to see Before and After side-by-side (à la [Octane cheat sheet](https://ember-learn.github.io/ember-octane-vs-classic-cheat-sheet/))

> <!-- replace with topic -->

Reviewer: ```suggestion
### Feedback for Ember Guides (IL, 10 min)

In the Ember Guides, we use the `+` and `-` to indicate what code have been added or deleted. For example, in the [Super Rentals Tutorial](https://guides.emberjs.com/release/tutorial/part-1/orientation/#toc_editing-files-and-live-reload) (and other pages), we might see,

<img width="480" alt="Screen Shot 2020-09-16 at 4 05 14 PM" src="https://user-images.githubusercontent.com/16869656/93392480-7995bc00-f836-11ea-953e-d0161a7538f8.png">

A developer provided me feedback that 2 interns that they were mentoring weren't sure how to read such code. The developer explained to me that the interns weren't yet familiar with diff tools that we might see on GitHub.

For more complex code (think about a large code review that you might have had to do), I think it's also possible that showing many changes with `+` or `-` may not be the best way to convey the changes.

I wondered if we can **update the code block component** to either:

- Provide a help button that, when clicked, explains how to read code with `+` and `-`
- Provide an option to see Before and After side-by-side (à la [Octane cheat sheet](https://ember-learn.github.io/ember-octane-vs-classic-cheat-sheet/))
```

- [ ] Jessica Jordan (JJ)
- [ ] Kenneth Larsen (KL)
- [ ] Melanie Sumner (MS)
- [ ] Ricardo Mendes (RM)
- [ ] Robert Wagner (RW)

### Guests
- [] Your name here (INITIALS)

## Weekly review

### Review action items (weekly)
See: [Action items board](https://github.com/orgs/ember-learn/projects/47)

### Review Initiatives board (every other week)
See: [Initiatives board](https://github.com/orgs/ember-learn/projects/19)

### Review PR queue
See: [tracking sheet](https://docs.google.com/spreadsheets/d/1sPyN9z9wZMpTNwqCfa6R9QSPZkIW4iQd-H4gZC7ILLk/edit#gid=2035777454)

## Topics

<!-- If you would like to add a topic to the agenda please add a suggestion to the PR using the following format: -->
<!-- ### Your topic (INITIALS, expected duration in minutes) -->
<!-- replace with topic -->
<!-- replace with topic -->
<!-- replace with topic -->
<!-- replace with topic -->
### Feedback for Ember Guides (IL, 10 min)

In the Ember Guides, we use the `+` and `-` to indicate what code have been added or deleted. For example, in the [Super Rentals Tutorial](https://guides.emberjs.com/release/tutorial/part-1/orientation/#toc_editing-files-and-live-reload) (and other pages), we might see,

> <img width="480" alt="Screen Shot 2020-09-16 at 4 05 14 PM" src="https://user-images.githubusercontent.com/16869656/93392480-7995bc00-f836-11ea-953e-d0161a7538f8.png">

Reviewer: ```suggestion
<img width="480" alt="The Ember Guides uses plus and minus symbols to show code changes" src="https://user-images.githubusercontent.com/16869656/93392480-7995bc00-f836-11ea-953e-d0161a7538f8.png">
```

> A developer provided me feedback that 2 interns that they were mentoring weren't sure how to read such code. The developer explained to me that the interns weren't yet familiar with diff tools that we might see on GitHub.

Reviewer: ```suggestion
A developer provided me feedback that 2 interns that they were mentoring weren't sure how to read such code. The developer explained to me that the interns weren't yet familiar with diff tools that we might regularly use on GitHub.
```

# September 17, 2020

Facilitator & Note Taker: 

## Attendees

Add yourself to the list if you attend and check the box!

### Core Team
- [ ] Amy Lam (AL)
- [x] Chris Manson (CM)
- [x] Isaac Lee (IL)
- [x] Jared Galanis (JG)
- [ ] Jen Weber (JW)
- [ ] Jessica Jordan (JJ)
- [x] Kenneth Larsen (KL)
- [ ] Melanie Sumner (MS)
- [x] Ricardo Mendes (RM)
- [ ] Robert Wagner (RW)

### Guests
- [x] Dan Monroe (DM) 

> - [] Your name here (INITIALS)

Reviewer: ```suggestion
- [x] James Davis (JD)
```

## Weekly review

### Review action items (weekly)
See: [Action items board](https://github.com/orgs/ember-learn/projects/47)

### Review Initiatives board (every other week)
See: [Initiatives board](https://github.com/orgs/ember-learn/projects/33)

### Review PR queue
See: [tracking sheet](https://docs.google.com/spreadsheets/d/1sPyN9z9wZMpTNwqCfa6R9QSPZkIW4iQd-H4gZC7ILLk/edit#gid=2035777454)

## Topics

> <!-- If you would like to add a topic to the agenda please add a suggestion to the PR using the following format: -->
> <!-- ### Your topic (INITIALS, expected duration in minutes) -->
> <!-- replace with topic -->
> <!-- replace with topic -->
> <!-- replace with topic -->
> <!-- replace with topic -->
> 

Reviewer: ```suggestion
```
